docs/javascript/v2/changelog/release-notes.mdx

```diff
@@ -15,6 +15,18 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
 
+## 2023-09-13
+Released: `@100mslive/hms-video-store@0.10.15`, `@100mslive/react-sdk@0.8.15`, `@100mslive/hls-player@0.1.15`, `@100mslive/roomkit-react@0.1.6`
+
+### Important
+
+We released Prebuilt. Customize your app themes and add video conferencing to your app with one line of code.
```

Comment:
Prebuilt was also available before. This version now supports changing themes and behaviour using dashboard.